src/projects/energiecheckpoint.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: layouts/project.liquid
+tags: ["Gebäude", "Energiewende","Verbrauch","Klimaschutz" ]
+id: energiecheckpoint
+title: 'Berliner EnergieCheckpoint'
```

Comment:
Could you add a [soft hyphen](https://en.wikipedia.org/wiki/Soft_hyphen) (`&shy;`) after the "Energie"? This will make sure that the long word wraps properly when there is not enough space available.

Here's an example for the Weihnachtsmarktfinder:

```plain
Berliner Weihnachts&shy;markt-Finder
```